index.bs

```diff
@@ -766,6 +767,11 @@ These events bubble up to SpeechSynthesis.
   No guarantee is given as to where charIndex will be with respect to word boundaries (such as at the end of the previous word or the beginning of the next word), only that all text before charIndex has already been spoken, and all text after charIndex has not yet been spoken.
   The user agent must return this value if the speech synthesis engine supports it, otherwise the user agent must return undefined.</dd>
 
+  <dt><dfn attribute for=SpeechSynthesisEvent>charLength</dfn> attribute</dt>
+  <dd>This attribute indicates the length of the text (word or sentence) that will be spoken corresponding to this event.
+  This attribute is the length, in characters, starting from this event's {{charIndex}}.
+  The user agent must return this value if the speech synthesis engine supports it or the user agent can otherwise determine it, otherwise the user agent must return undefined.</dd>
```

Comment:
It's actually not possible to "return undefined" since the type is unsigned long. This needs to be fixed here, above and below before merging this PR.

Reply:
Most implementations (Firefox, Chrome) seem to return the default value as defined by SpeechSynthesisEventInit. Would it suffice to change the wording to match that behavior (default of 0 for charLength), or would it be more appropriate to change the type to be nullable?